Crash

- Yes, I am running the latest version
- Yes, I made sure this issue doesn't persist without this mod
- Yes, I checked to make sure that this issue has not already been filed

Randomly crashes during a heist.

Using another HUD atm.

* Operating System: Windows 10 version 1903
* BLT version: SuperBLT 1.1.5.1
* Other mods installed:
* Auto-Fire Sound Fix, BeardLib, Better Bots, Bot Weapons and Equipment, Drag and Drop Inventory, Goonmod's Custom Waypoints, Iter, Keepers, Lobby Player Info, More Weapons Stats, No Duplicated Bullets, BLT+, Cheater Kicker, Color Grading Changer, CustomFOV, Extended Continental Coin Shop, Improved Offline Funcionality, Kill Streak, PAYDAY: The Heist Equipment, Perfect Viewmodel, Simple Crosshair, StatTrak, Stoic Slurp, Tweakdata Fixed, No Flashbang Ringing, PAYDAY: The Heist HUD, Restructured Menus, Slides & Bolts Lock Back, The Fixes.

## BLT log

10:15:38 AM Lua: [Error] Mod image at path does not exist! mods/No Duplicated Bullets/tdlq.png
10:15:38 AM Lua: [BeardLib] [Add Framework] Initialized Mod: assets/mod_overrides/Roller Mobster/
10:15:38 AM Lua: [BeardLib] [Base Framework] Initialized Mod: mods/PerfectViewModel/
10:15:38 AM Lua: [BeardLib] [Base Framework] Initialized Mod: mods/StatTrak/
10:15:38 AM Lua: [BotWeapons] Setting up weapons
10:15:38 AM Lua: [BotWeapons] Fixing gang presets
10:15:38 AM Lua: [stattrak][ERROR] [WeaponMod] Weapon wpn_fps_bow_predator_pts does not exist. Cannot add part wpn_fps_trk_stattrak to it.
10:15:38 AM Lua: [GageModShop] Loaded options
10:15:38 AM Lua: Loading mod settings : nil
10:15:39 AM FATAL ERROR:  (c:\users\znix\source\repos\superblt\src\initiatestate.cpp:240) mods/base/req/utils/json-1.0.lua:212: Failed to scan constant from string {
	"custom_FOV_menu_title" : "Custom FOV",
	"custom_FOV_menu_desc" : "Customize your FOV",

	"sld_more_FOV_title" : "FOV",
	"sld_more_FOV_desc" : "Customize your FOV",

}
 at starting position 170
stack traceback:
	[C]: in function 'assert'
	mods/base/req/utils/json-1.0.lua:212: in function 'decode'
	mods/base/req/utils/json-1.0.lua:262: in function 'decode'
	mods/base/req/utils/json.lua:9: in function <mods/base/req/utils/json.lua:8>
	[C]: in function 'pcall'
	mods/base/req/utils/json.lua:8: in function 'decode'
	mods/base/lua/LocalizationManager.lua:68: in function 'load_localization_file'
	mods/CustomFOV/lua/init.lua:20: in function 'func'
	mods/base/req/core/Hooks.lua:109: in function 'Call'
	mods/base/lua/LocalizationManager.lua:8: in function 'origfunc'
	mods/The Fixes/localizationmanager.lua:19: in function 'init'
	[string "core/lib/utils/coreclass.lua"]:35: in function 'new'
	[string "core/lib/setups/coresetup.lua"]:343: in function <[string "core/lib/setups/coresetup.lua"]:281>
10:15:39 AM Lua: Found a json error, attempting to use old json loader!
10:15:39 AM FATAL ERROR:  (c:\users\znix\source\repos\superblt\src\initiatestate.cpp:240) mods/base/req/utils/json-1.0.lua:212: Failed to scan constant from string {
	"custom_FOV_menu_title" : "Custom FOV",
	"custom_FOV_menu_desc" : "Customize your FOV",

	"sld_more_FOV_title" : "FOV",
	"sld_more_FOV_desc" : "Customize your FOV",

}
 at starting position 170
stack traceback:
	[C]: in function 'assert'
	mods/base/req/utils/json-1.0.lua:212: in function 'decode'
	mods/base/req/utils/json-1.0.lua:262: in function 'decode'
	mods/base/req/utils/json.lua:9: in function <mods/base/req/utils/json.lua:8>
	[C]: in function 'pcall'
	mods/base/req/utils/json.lua:8: in function 'decode'
	mods/base/lua/LocalizationManager.lua:68: in function 'load_localization_file'
	mods/CustomFOV/lua/init.lua:25: in function 'func'
	mods/base/req/core/Hooks.lua:109: in function 'Call'
	mods/base/lua/LocalizationManager.lua:8: in function 'origfunc'
	mods/The Fixes/localizationmanager.lua:19: in function 'init'
	[string "core/lib/utils/coreclass.lua"]:35: in function 'new'
	[string "core/lib/setups/coresetup.lua"]:343: in function <[string "core/lib/setups/coresetup.lua"]:281>
10:15:39 AM Lua: Found a json error, attempting to use old json loader!
10:15:39 AM Lua: [Keybind] Registered keybind [BLTKeybind CustomWaypointsPlaceWaypoint]
10:15:39 AM Lua: [Keybind] Registered keybind [BLTKeybind CustomWaypointsRemoveWaypoint]
10:15:39 AM Lua: [Keybind] Registered keybind [BLTKeybind CustomWaypointsPreviousWaypoint]
10:15:39 AM Lua: [Keybind] Registered keybind [BLTKeybind CustomWaypointsNextWaypoint]
10:15:39 AM Lua: [Keybind] Registered keybind [BLTKeybind stattrak_change_mode]
10:15:39 AM Lua: [Keybind] Registered keybind [BLTKeybind keepers_follow]
10:15:39 AM Lua: [Keybind] Registered keybind [BLTKeybind keepers_stay]
10:15:43 AM Lua: [4] 1st Lt. Germoviet: GNAP/Keepers?/My Joker
10:15:43 AM Lua: Steam RP updated: Playing:  Cook Off (MAYHAM)
10:15:43 AM Lua: [4] Shapechanger: GNAP/BCO/ flm white_black facepaint_skull matteblack none wpn_fps_lmg_par 3_4_7_6_25_22_1_18_30_ wpn_fps_smg_baka 1_2_3_4_22_26_7_ taser par_same-3-0 !-1-0|1.0
10:15:44 AM Lua: [4] Shapechanger: GNAP/BCO/ flm white_black facepaint_skull matteblack none wpn_fps_lmg_par 3_4_7_6_25_22_1_18_30_ wpn_fps_smg_baka 1_2_3_4_22_26_7_ taser par_same-3-0 !-1-0|1.0
10:15:44 AM Lua: [4] Shapechanger: GNAP/BCO/ flm white_black facepaint_skull matteblack none wpn_fps_lmg_par 3_4_7_6_25_22_1_18_30_ wpn_fps_smg_baka 1_2_3_4_22_26_7_ taser par_same-3-0 !-1-0|1.0
10:15:44 AM Lua: [4] Shapechanger: GNAP/BCO/ flm white_black facepaint_skull matteblack none wpn_fps_lmg_par 3_4_7_6_25_22_1_18_30_ wpn_fps_smg_baka 1_2_3_4_22_26_7_ taser par_same-3-0 !-1-0|1.0
10:15:44 AM Lua: [4] Shapechanger: GNAP/BCO/ flm white_black facepaint_skull matteblack none wpn_fps_lmg_par 3_4_7_6_25_22_1_18_30_ wpn_fps_smg_baka 1_2_3_4_22_26_7_ taser par_same-3-0 !-1-0|1.0
10:15:44 AM Lua: [4] Shapechanger: GNAP/BCO/ flm white_black facepaint_skull matteblack none wpn_fps_lmg_par 3_4_7_6_25_22_1_18_30_ wpn_fps_smg_baka 1_2_3_4_22_26_7_ taser par_same-3-0 !-1-0|1.0
10:15:46 AM Lua: [4] Shapechanger: GNAP/BCO/ flm white_black facepaint_skull matteblack none wpn_fps_lmg_par 3_4_7_6_25_22_1_18_30_ wpn_fps_smg_baka 1_2_3_4_22_26_7_ taser par_same-3-0 !-1-0|1.0
10:15:46 AM Lua: [4] Shapechanger: GNAP/BCO/ flm white_black facepaint_skull matteblack none wpn_fps_lmg_par 3_4_7_6_25_22_1_18_30_ wpn_fps_smg_baka 1_2_3_4_22_26_7_ taser par_same-3-0 !-1-0|1.0
10:15:47 AM Lua: [1] L Y N X: rip our ass
10:15:51 AM Lua: [1] 1st Lt. Germoviet: yep
10:15:57 AM Lua: [1] SYSTEM: L Y N X left.
10:15:57 AM Lua: Steam RP updated: Playing:  Cook Off (MAYHAM)
10:16:01 AM Lua: [1] Shapechanger: Imagine being so entitled that you fuck over not just the guy giving you lip
10:16:06 AM Lua: [1] Shapechanger: But two randos as well
10:16:10 AM Lua: [1] Shapechanger: Fucking kek
10:16:11 AM Lua: [1] SYSTEM: Shapechanger unlocked asset: Ammo Bag.
10:16:12 AM Lua: [1] SYSTEM: Shapechanger unlocked asset: Grenade Case.
10:16:13 AM Lua: [1] SYSTEM: Shapechanger unlocked asset: Floodlights.
10:16:14 AM Lua: [1] SYSTEM: Shapechanger unlocked asset: Doctor Bag.
10:16:15 AM Lua: [1] SYSTEM: Shapechanger unlocked asset: Vantage Point.
10:16:16 AM Lua: [1] 1st Lt. Germoviet: next time just ban him
10:16:25 AM Lua: [1] Shapechanger: Yep
10:16:30 AM Lua: [4] Shapechanger: GNAP/BCO/ flm white_black facepaint_skull matteblack none wpn_fps_lmg_par 3_4_7_6_25_22_1_18_30_ wpn_fps_smg_baka 1_2_3_4_22_26_7_ taser par_same-3-0 !-1-0|1.0
10:16:30 AM Lua: [4] Shapechanger: GNAP/BSEW/
10:16:30 AM Lua: [4] Shapechanger: GNAP/BCO/ flm white_black facepaint_skull matteblack none wpn_fps_lmg_par 3_4_7_6_25_22_1_18_30_ wpn_fps_smg_baka 1_2_3_4_22_26_7_ taser par_same-3-0 !-1-0|1.0
10:16:30 AM Lua: [4] Shapechanger: GNAP/BCO/ flm white_black facepaint_skull matteblack none wpn_fps_lmg_par 3_4_7_6_25_22_1_18_30_ wpn_fps_smg_baka 1_2_3_4_22_26_7_ taser par_same-3-0 !-1-0|1.0
10:16:30 AM Lua: [4] Shapechanger: GNAP/BCO/ flm white_black facepaint_skull matteblack none wpn_fps_lmg_par 3_4_7_6_25_22_1_18_30_ wpn_fps_smg_baka 1_2_3_4_22_26_7_ taser par_same-3-0 !-1-0|1.0
10:16:30 AM Lua: Steam RP updated: Playing:  Cook Off (MAYHAM)
10:16:35 AM Lua: [4] Shapechanger: GNAP/BSEW/
10:16:37 AM Lua: [4] Shapechanger: GNAP/BSEW/
10:16:37 AM Lua: [4] Shapechanger: GNAP/BSEW/

## Crash log(s)

Application has crashed: C++ exception
mods/WolfHUD-master/lua/RichPresence.lua:26: attempt to index field 'lobby_handler' (a nil value)



SCRIPT STACK

refresh_rich_presence() lib/managers/platformmanager.lua:81
remove_peer() lib/network/base/basenetworksession.lua:348
remove_peer() lib/network/base/clientnetworksession.lua:644
on_peer_left() lib/network/base/basenetworksession.lua:529
lib/network/base/handlers/connectionnetworkhandler.lua:260


-------------------------------

Callstack:

         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     zip_get_name                                        
         payday2_win32_release  (???)     zip_get_name                                        


-------------------------------

Current thread: Main

-------------------------------

System information:
	Application version : 1.92.790
	CPU : AMD Ryzen 3 1200 Quad-Core Processor (2 cores); SSE; SSE2; SSE3; SSSE3; SSE4.1
	DirectX : 12.0 
	GPU : NVIDIA GeForce GTX 1050 Ti / nvldumd.dll[26.21.14.3615]
	Language : english
	Memory :     16335MB 552KB
	OS : 6.2.9200 () 0x100-0x1 (64 bits)
	Physics : threaded
	Renderer : DX9 threaded
	Sound : Realtek Semiconductor Corp. (Altavoces (Realtek High Definition Audio))

--- a/lua/RichPresence.lua
+++ b/lua/RichPresence.lua
@@ -22,7 +22,7 @@ if RequiredScript == "lib/managers/platformmanager" then
 					game_state = "private"
 				else
 					-- Handle Steam RP Grouping
-					if not Global.game_settings.single_player then
+					if not Global.game_settings.single_player and managers.network.matchmake.lobby_handler ~= nil then
 						group_key = managers.network.matchmake.lobby_handler:id()
 
 						local session = managers.network:session()

